monai/handlers/transform_inverter.py

```diff
@@ -104,7 +104,11 @@ def __call__(self, engine: Engine) -> None:
 
             transform_info = engine.state.batch[transform_key]
             if nearest_interp:
-                convert_inverse_interp_mode(trans_info=transform_info, mode="nearest", align_corners=None)
+                transform_info = convert_inverse_interp_mode(
+                    trans_info=deepcopy(transform_info),
```

Comment:
Maybe several output_keys share the same batch_key during inverting, for example, all items use the transform_info of `label`, should deepcopy if we modify it.

Reply:
thanks, we agreed that we should create some followup test cases